test/slider-test.jsx

```diff
@@ -530,115 +530,145 @@ describe('Slider API', () => {
     });
   });
 
-  /*
-    Note: Skipping this test because it requires the handle's ref to
-    be executed, which requires a mount, as well as access to the instance
-    methods of slider. Please feel free to re-write or remove test, as desired.
-    TODO: Philip and Maja
-  */
-  describe.skip('getNextState', () => {
-    it('should return the next state given a position and index', () => {
-      const slider = mount(<Slider values={[0]} />).instance();
-      const nextState = slider.getNextState(0, 50);
-      assert(nextState.handlePos[0] === 50, 'handle is at 50%');
-      assert(nextState.values[0] === 50, 'the value is 50');
-    });
+  describe('need slider bounding box defined', () => {
+    beforeEach(() => {
+      // This return value for getSliderBoundingBox is taken from the actual
+      // in-browser values of one of the examples.
+      sinon
+        .stub(PureSlider.prototype, 'getSliderBoundingBox')
```

Comment:
Since `getSliderBoundingBox` is never cached as a reference, I don't think we need to export the pure component at all - you could move the stub to be `sinon.stub(slider, 'getSliderBoundingBox')` after the shallow call, and it'd work fine.